gcc/rust/checks/errors/rust-feature-gate.cc

```diff
@@ -105,4 +126,61 @@ FeatureGate::visit (AST::ExternBlock &block)
     }
 }
 
+void
+FeatureGate::check_rustc_attri (const std::vector<AST::Attribute> &attributes)
+{
+  std::for_each (attributes.begin (), attributes.end (),
```

Comment:
is std::for_each c++11? https://en.cppreference.com/w/cpp/algorithm/for_each

You will likely need to change do a plain range based for loop ```for (auto &attr : attributes)` is fine`

Reply:
yeah. I forgot the c++11 restrict.